### PR TITLE
Fix excessive "replacing bundled versions" bug

### PR DESCRIPTION
**Pull request by iarna**
These are terrible. Everything is terrible. Let's patch this out maybe?

### CODE REVIEW

File: lib/install/action/extract.js [hunk 1]
```diff
@@ -2,7 +2,7 @@
 var path = require('path')
 var iferr = require('iferr')
 var asyncMap = require('slide').asyncMap
-var rename = require('graceful-fs').rename
+var fs = require('graceful-fs')
```
Comment (hemanth) [inline lib/install/action/extract.js]: Isn't `var rename = require('graceful-fs').rename` lighter?

**Comment by othiym23** [comment]
Landed as e05e2961ed125abf39da895d9c851650a4485e2d, along with some test cleanup in d804c3c662cebcc0dc0ca43dfc0b5c64c6d7f96e to get the tests passing. Thanks, Rebecca!

**Comment by fengmk2** [comment]
Can we add a test case for this bugfix?

**Comment by othiym23** [comment]
It works in concert with the tests added to `npm@3.6"5.0` for the bundling bug, but we'd welcome more tests around this behavior for sure!

**Pull request closed without merging**